monai/handlers/probability_maps.py

```diff
@@ -91,8 +92,8 @@ def __call__(self, engine: Engine) -> None:
         """
```

Comment:
As this handler can only work with `MaskedPatchWSIDataset` and `SlidingPatchWSIDataset`, and must work with MetaTensor now, please highlight it in the docstring of the class.

Thanks.